addons/im_livechat/models/im_livechat_channel.py

```diff
@@ -103,23 +102,15 @@ def _compute_nbr_channel(self):
     def action_join(self):
         self.ensure_one()
         self.user_ids = [Command.link(self.env.user.id)]
-        self.env["bus.bus"]._sendone(
-            self.env.user.partner_id,
-            "mail.record/insert",
-            Store(
-                "LivechatChannel", {"id": self.id, "name": self.name, "hasSelfAsMember": True}
-            ).get_result(),
+        self.user._bus_send_store(
```

Comment:
```suggestion
        self.env.user._bus_send_store(
```

Reply:
Good catch, I checked 3 times and still missed that one.